.vscode/settings.json

```diff
@@ -83,5 +83,8 @@
     "subspec",
     "uifabricshared",
     "unemphasized"
-  ]
+  ],
+  "editor.defaultFormatter": "esbenp.prettier-vscode",
+  "prettier.configPath": "E:\\fluentui-react-native\\prettier.config.js",
```

Comment:
hardcoded path?

Reply:
Whoops I'll revert the changes to this file.